herd/libdir/asl-pseudocode/implementations0.asl

```diff
@@ -38,5 +38,7 @@ PhysMemRetStatus PhysMemWrite(AddressDescriptor desc, integer size, AccessDescri
 
 (PhysMemRetStatus, bits(8*size)) PhysMemRead(AddressDescriptor desc, integer size,
                                              AccessDescriptor accdesc)
+    // DEBUG(accdesc.acctype, desc.paddress.address);
```

Comment:
nit: are changes to this file intentional? Similarly for `// DEBUG` in `patches-vmsa.asl`